piety-promoted-v1/modernized/02-piety-promoted-v1.adoc

```diff
@@ -360,12 +363,15 @@ Given forth by me, who am known to men by the name of Marmaduke Stevenson;
 but have a new name given me, which the world knows not of,
 written in the Book of Life.`'
```

Comment:
remember to look through these embedded letters or papers, etc. and REMOVE the quotation marks [in this case they are apostrophes] before and after paragraphs. Because we are embedding them, they don't need to be wrapped in quotes.

Reply:
Yes, I thought of that too late too, and started doing them, but left some to go back to later and I now just remembered that I can't go back and fix things at the end after I've submitted them  like in an ODT. Duh. Haha